libraries/status.rb

```diff
@@ -18,6 +18,8 @@ def name
 
   def id
     current_value['id']
+  rescue
```

Comment:
You should only rescue the specific error class here - you don't want to do this for all exceptions.